.github/workflows/release.yml

```diff
@@ -0,0 +1,181 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+      - "nightly"
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  windows-build:
+    name: "Build on Windows with Spinnaker SDK"
+    runs-on:
+      - self-hosted
+      - spinnaker
+      - BFLY-U3-23S6M-C # for tests
+      - windows
+
+    permissions:
+      actions: write
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Enable long paths
+        run: |
+          git config --global core.longpaths true
+        shell: pwsh
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Get CMake 3.24
+        uses: lukka/get-cmake@latest
+        with:
+          useCloudCache: false
+          useLocalCache: true
+          cmakeVersion: 3.24.3
+
+      - name: Install MSVC
+        uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: amd64_x86
+
+      - name: Build
+        run: |
+          cmake -B ${{github.workspace}}/test -DCMAKE_BUILD_TYPE=Release
+          cmake --build ${{github.workspace}}/test --config Release
+
+      - name: Test  # don't release unless tests pass
+        working-directory: ${{github.workspace}}/test
+        run: |
+          ctest -C Release -L anyplatform --output-on-failure
```

Comment:
I think this is just to run the acquire-core-libs unit tests. Do we really need to run these here?